### PR TITLE
Testing resource table

### PR DESCRIPTION
**Pull request by LiorRey**
Related to #69 

Added file 'test_resource.py' to 'tests' folder, for testing the addition of a new resource.

### CODE REVIEW

File: tests/test_resource.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from tutor.models.course import Resource
+
+
+def test_add_new_resource():
+    client = app.test_client()
+    assert Resource.query.all() == []
+    response = client.post('/resource', data=dict(
+        title='test_resource_title'
+        content='test_resource_content'
+        link='test_resource_link'))
```
Comment (mosesnagar) [inline tests/test_resource.py:10]: You should add `,` to separate between the properties.
```
data=dict(
        title='test_resource_title',
        content='test_resource_content',
        link='test_resource_link')
```

File: tests/test_resource.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from tutor.models.course import Resource
+
+
+def test_add_new_resource():
+    client = app.test_client()
+    assert Resource.query.all() == []
+    response = client.post('/resource', data=dict(
```
Comment (mosesnagar) [inline tests/test_resource.py:7]: There is no '/resource` route. If you want to add a new resource you should call `/resource/new?id=%COURSE_ID%. As on `/tutor/routes/resoruces.py`. 
By the way, that kind of test already implemented on #62 . Maybe you can focus on testing the model itself, as I did on `Model` test, instead the route?

File: tests/test_resource.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+from tutor.models.course import Resource
+
+
+def test_add_new_resource():
```
Comment (bardielle) [inline tests/test_resource.py:4]: As @ifireball mention in the last session - what are you trying to test? it seems that you are testing the ORM.
This is a unit testing- you don't need to test ORM, it's not yours and someone else already tested it 

remove that test and add a real UT

Comment (LiorRey) [reply tests/test_resource.py:4]: I tried to test the addition of a new resource. Is this what you meant in "testing the ORM"?

Comment (yuvalmargules) [reply tests/test_resource.py:4]: There's already a merged test_source.py at which I think contains the code on this PR(by looking at the conflict). IMHO I don't see a reason to continue this one. please correct me if I'm wrong.

**Comment by yuvalmargules** [comment]
There's already a merged test_source.py at which I think contains the code on this PR(by looking at the conflict). IMHO I don't see a reason to continue this one. please correct me if I'm wrong.

**Pull request closed without merging**